### ISSUE TITLE
Update handling of VOD queries to accept more arguments

### ISSUE THREAD

**Issue by coltatgh** [Jyckle/sncrs#13]
Complements SmashNightBot issue for updating !vods command to accept additional arguments: [https://github.com/coltatgh/SmashNightBot/issues/7](url)

Filters would be needed to select VODs
- Between 2 given players
- For a given player (or pair of players) on a given SmashNight
- For a given player during specific SmashNight season

**Comment by Jyckle** [comment]
@coltatgh Hey, what do you think is the desired method of specifying a given smashnight?
Potential options:
- By date
- By season/number
- By some other thing I'm not thinking of

**Comment by Jyckle** [comment]
Had another thought, might leave it querying based on Smashnight I'd in the API, and work on a separate Smashnight API, so we can prompt the user with which Smashnight they want, and then use the id from that in this query! 

I.e.
```
User: !vods by_sn player1 player2
Which Smashnight would you like vods from?
1) Smashnight Season 5 Night 1 on 2023-02-01
2) Smashnight Season 4 Night 10 on 2022-10-01
Enter the number: <User inputs 2>
Your vods from Smashnight Season 4 Night 10 on 2022-10-01 are:
- list of videos between player 1 and player 2
```

**Comment by coltatgh** [comment]
Copying our IM discussion here for future reference: I think specifying the SmashNight in the format #.# (e.g. 5.2) should be sufficient

**Comment by Jyckle** [comment]
Closed with PR #17
